lib/src/core/signal_client.dart

```diff
@@ -116,8 +118,6 @@ class SignalClient extends Disposable with EventsEmittable<SignalEvent> {
 
       if (_connectivityResult.contains(ConnectivityResult.none)) {
         logger.warning('no internet connection');
-        events.emit(SignalDisconnectedEvent(
```

Comment:
There is no need to emit a SignalDisconnectedEvent event here, because connect or resumeConnect/restartConnect will catch the ConnectException and take the correct follow-up action.